2025-01-23 01:58:47,490 WARNING werkzeug MainThread :  * Debugger is active!
2025-01-23 01:58:47,497 INFO werkzeug MainThread :  * Debugger PIN: 846-340-584
2025-01-23 01:58:49,276 INFO main Thread-2 (process_request_thread) : ui
2025-01-23 01:58:49,279 INFO werkzeug Thread-2 (process_request_thread) : 127.0.0.1 - - [23/Jan/2025 01:58:49] "GET /ui HTTP/1.1" 200 -
2025-01-23 01:58:49,344 INFO werkzeug Thread-3 (process_request_thread) : 127.0.0.1 - - [23/Jan/2025 01:58:49] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -
2025-01-23 01:58:49,791 INFO main Thread-4 (process_request_thread) : func_2
2025-01-23 01:58:49,807 INFO werkzeug Thread-4 (process_request_thread) : 127.0.0.1 - - [23/Jan/2025 01:58:49] "POST /func_2 HTTP/1.1" 200 -
2025-01-23 02:01:51,380 INFO main Thread-6 (process_request_thread) : ui
2025-01-23 02:01:51,381 INFO werkzeug Thread-6 (process_request_thread) : 127.0.0.1 - - [23/Jan/2025 02:01:51] "GET /ui HTTP/1.1" 200 -
2025-01-23 02:01:51,443 INFO werkzeug Thread-7 (process_request_thread) : 127.0.0.1 - - [23/Jan/2025 02:01:51] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -
2025-01-23 02:01:51,878 INFO main Thread-8 (process_request_thread) : func_2
2025-01-23 02:01:51,890 INFO werkzeug Thread-8 (process_request_thread) : 127.0.0.1 - - [23/Jan/2025 02:01:51] "POST /func_2 HTTP/1.1" 200 -
2025-01-23 02:03:37,384 INFO main Thread-10 (process_request_thread) : ui
2025-01-23 02:03:37,384 INFO werkzeug Thread-10 (process_request_thread) : 127.0.0.1 - - [23/Jan/2025 02:03:37] "GET /ui HTTP/1.1" 200 -
2025-01-23 02:03:37,567 INFO werkzeug Thread-11 (process_request_thread) : 127.0.0.1 - - [23/Jan/2025 02:03:37] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -
2025-01-23 02:03:37,890 INFO main Thread-12 (process_request_thread) : func_1
2025-01-23 02:03:37,891 INFO werkzeug Thread-12 (process_request_thread) : 127.0.0.1 - - [23/Jan/2025 02:03:37] "GET /func_1?idx=2 HTTP/1.1" 200 -
2025-01-23 02:03:49,758 INFO main Thread-14 (process_request_thread) : ui
2025-01-23 02:03:49,758 INFO werkzeug Thread-14 (process_request_thread) : 127.0.0.1 - - [23/Jan/2025 02:03:49] "GET /ui HTTP/1.1" 200 -
2025-01-23 02:03:49,820 INFO werkzeug Thread-15 (process_request_thread) : 127.0.0.1 - - [23/Jan/2025 02:03:49] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -
2025-01-23 02:03:50,309 INFO main Thread-16 (process_request_thread) : func_2
2025-01-23 02:03:50,326 INFO werkzeug Thread-16 (process_request_thread) : 127.0.0.1 - - [23/Jan/2025 02:03:50] "POST /func_2 HTTP/1.1" 200 -
2025-01-23 02:03:50,840 INFO main Thread-17 (process_request_thread) : ui
2025-01-23 02:03:50,841 INFO werkzeug Thread-17 (process_request_thread) : 127.0.0.1 - - [23/Jan/2025 02:03:50] "GET /ui HTTP/1.1" 200 -
2025-01-23 02:03:51,275 INFO main Thread-18 (process_request_thread) : func_2
2025-01-23 02:03:51,279 INFO main Thread-18 (process_request_thread) : Traceback (most recent call last):
  File "C:\code\PycharmProjects\song\server\main.py", line 96, in func_2
    db.Database.execute(Query.post_insert(val))
  File "C:\code\PycharmProjects\song\server\db_f\db.py", line 34, in execute
    self.cursor.execute(query, args)
  File "C:\Users\¼Û¹Î±Ô\AppData\Local\Programs\Python\Python312\Lib\site-packages\pymysql\cursors.py", line 153, in execute
    result = self._query(query)
             ^^^^^^^^^^^^^^^^^^
  File "C:\Users\¼Û¹Î±Ô\AppData\Local\Programs\Python\Python312\Lib\site-packages\pymysql\cursors.py", line 322, in _query
    conn.query(q)
  File "C:\Users\¼Û¹Î±Ô\AppData\Local\Programs\Python\Python312\Lib\site-packages\pymysql\connections.py", line 558, in query
    self._affected_rows = self._read_query_result(unbuffered=unbuffered)
                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\¼Û¹Î±Ô\AppData\Local\Programs\Python\Python312\Lib\site-packages\pymysql\connections.py", line 822, in _read_query_result
    result.read()
  File "C:\Users\¼Û¹Î±Ô\AppData\Local\Programs\Python\Python312\Lib\site-packages\pymysql\connections.py", line 1200, in read
    first_packet = self.connection._read_packet()
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\¼Û¹Î±Ô\AppData\Local\Programs\Python\Python312\Lib\site-packages\pymysql\connections.py", line 772, in _read_packet
    packet.raise_for_error()
  File "C:\Users\¼Û¹Î±Ô\AppData\Local\Programs\Python\Python312\Lib\site-packages\pymysql\protocol.py", line 221, in raise_for_error
    err.raise_mysql_exception(self._data)
  File "C:\Users\¼Û¹Î±Ô\AppData\Local\Programs\Python\Python312\Lib\site-packages\pymysql\err.py", line 143, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.DataError: (1264, "Out of range value for column 'input_1' at row 1")

2025-01-23 02:03:51,279 INFO werkzeug Thread-18 (process_request_thread) : 127.0.0.1 - - [23/Jan/2025 02:03:51] "POST /func_2 HTTP/1.1" 200 -
2025-01-23 02:03:51,673 INFO main Thread-19 (process_request_thread) : ui
2025-01-23 02:03:51,674 INFO werkzeug Thread-19 (process_request_thread) : 127.0.0.1 - - [23/Jan/2025 02:03:51] "GET /ui HTTP/1.1" 200 -
2025-01-23 02:03:51,987 INFO main Thread-20 (process_request_thread) : func_1
2025-01-23 02:03:51,988 INFO werkzeug Thread-20 (process_request_thread) : 127.0.0.1 - - [23/Jan/2025 02:03:51] "GET /func_1?idx=2 HTTP/1.1" 200 -
2025-01-23 02:03:52,386 INFO main Thread-21 (process_request_thread) : ui
2025-01-23 02:03:52,386 INFO werkzeug Thread-21 (process_request_thread) : 127.0.0.1 - - [23/Jan/2025 02:03:52] "GET /ui HTTP/1.1" 200 -
2025-01-23 02:03:52,688 INFO main Thread-22 (process_request_thread) : func_1
2025-01-23 02:03:52,689 INFO main Thread-22 (process_request_thread) : Traceback (most recent call last):
  File "C:\code\PycharmProjects\song\server\main.py", line 57, in func_1
    a['data'] = db.Database.execute(Query.get_select_one(val))
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\code\PycharmProjects\song\server\db_f\db.py", line 38, in execute
    raise ValueError('Index not found')
ValueError: Index not found

2025-01-23 02:03:52,690 INFO werkzeug Thread-22 (process_request_thread) : 127.0.0.1 - - [23/Jan/2025 02:03:52] "GET /func_1?idx=-2147483649 HTTP/1.1" 200 -
2025-01-23 02:03:53,087 INFO main Thread-23 (process_request_thread) : ui
2025-01-23 02:03:53,088 INFO werkzeug Thread-23 (process_request_thread) : 127.0.0.1 - - [23/Jan/2025 02:03:53] "GET /ui HTTP/1.1" 200 -
2025-01-23 02:03:53,550 INFO main Thread-24 (process_request_thread) : func_1
2025-01-23 02:03:53,556 INFO werkzeug Thread-24 (process_request_thread) : 127.0.0.1 - - [23/Jan/2025 02:03:53] "POST /func_1 HTTP/1.1" 200 -
2025-01-23 02:03:53,956 INFO main Thread-25 (process_request_thread) : ui
2025-01-23 02:03:53,957 INFO werkzeug Thread-25 (process_request_thread) : 127.0.0.1 - - [23/Jan/2025 02:03:53] "GET /ui HTTP/1.1" 200 -
2025-01-23 02:03:54,415 INFO main Thread-26 (process_request_thread) : func_1
2025-01-23 02:03:54,416 INFO main Thread-26 (process_request_thread) : Traceback (most recent call last):
  File "C:\code\PycharmProjects\song\server\main.py", line 73, in func_1
    db.Database.execute(Query.get_select_one(int(val['idx'])))
  File "C:\code\PycharmProjects\song\server\db_f\db.py", line 38, in execute
    raise ValueError('Index not found')
ValueError: Index not found

2025-01-23 02:03:54,416 INFO werkzeug Thread-26 (process_request_thread) : 127.0.0.1 - - [23/Jan/2025 02:03:54] "POST /func_1 HTTP/1.1" 200 -
2025-01-23 02:03:54,806 INFO main Thread-27 (process_request_thread) : ui
2025-01-23 02:03:54,807 INFO werkzeug Thread-27 (process_request_thread) : 127.0.0.1 - - [23/Jan/2025 02:03:54] "GET /ui HTTP/1.1" 200 -
2025-01-23 02:03:55,108 INFO main Thread-28 (process_request_thread) : func_3
2025-01-23 02:03:55,108 INFO werkzeug Thread-28 (process_request_thread) : 127.0.0.1 - - [23/Jan/2025 02:03:55] "POST /func_3 HTTP/1.1" 200 -
2025-01-23 02:03:55,514 INFO main Thread-29 (process_request_thread) : ui
2025-01-23 02:03:55,514 INFO werkzeug Thread-29 (process_request_thread) : 127.0.0.1 - - [23/Jan/2025 02:03:55] "GET /ui HTTP/1.1" 200 -
2025-01-23 02:03:55,819 INFO main Thread-30 (process_request_thread) : func_3
2025-01-23 02:03:55,819 INFO werkzeug Thread-30 (process_request_thread) : 127.0.0.1 - - [23/Jan/2025 02:03:55] "POST /func_3 HTTP/1.1" 200 -
2025-01-23 02:03:56,209 INFO main Thread-31 (process_request_thread) : ui
2025-01-23 02:03:56,209 INFO werkzeug Thread-31 (process_request_thread) : 127.0.0.1 - - [23/Jan/2025 02:03:56] "GET /ui HTTP/1.1" 200 -
2025-01-23 02:03:56,522 INFO main Thread-32 (process_request_thread) : func_3
2025-01-23 02:03:56,522 INFO werkzeug Thread-32 (process_request_thread) : 127.0.0.1 - - [23/Jan/2025 02:03:56] "POST /func_3 HTTP/1.1" 200 -
2025-01-23 02:04:43,227 INFO main Thread-35 (process_request_thread) : default
2025-01-23 02:04:43,227 INFO werkzeug Thread-35 (process_request_thread) : 127.0.0.1 - - [23/Jan/2025 02:04:43] "GET / HTTP/1.1" 200 -
2025-01-23 02:04:43,271 INFO werkzeug Thread-36 (process_request_thread) : 127.0.0.1 - - [23/Jan/2025 02:04:43] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -
2025-01-23 02:04:44,249 INFO main Thread-37 (process_request_thread) : ui
2025-01-23 02:04:44,249 INFO werkzeug Thread-37 (process_request_thread) : 127.0.0.1 - - [23/Jan/2025 02:04:44] "GET /ui HTTP/1.1" 200 -
2025-01-23 02:04:47,718 INFO main Thread-38 (process_request_thread) : func_1
2025-01-23 02:04:47,719 INFO main Thread-38 (process_request_thread) : Traceback (most recent call last):
  File "C:\code\PycharmProjects\song\server\main.py", line 57, in func_1
    a['data'] = db.Database.execute(Query.get_select_one(val))
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\code\PycharmProjects\song\server\db_f\db.py", line 38, in execute
    raise ValueError('Index not found')
ValueError: Index not found

2025-01-23 02:04:47,719 INFO werkzeug Thread-38 (process_request_thread) : 127.0.0.1 - - [23/Jan/2025 02:04:47] "GET /func_1?idx=3 HTTP/1.1" 200 -
2025-01-23 02:05:37,958 INFO werkzeug MainThread :  * Detected change in 'C:\\code\\PycharmProjects\\test_script\\Selenium\\item\\element_list.py', reloading
